### ISSUE TITLE
[Kotlin] Add hexagon

### ISSUE THREAD

**Issue by waghanza** [the-benchmarker/web-frameworks#1770]
Hi @jaguililla,

Would you mind to add (or consent that we add) [hexagon](https://github.com/hexagonkt/hexagon) in `here` ?

This project compare performance based on 3 routes :

+ `GET` `/`, **SHOULD** return a successful status code and an empty string
+ `GET` `/user/:id`, **SHOULD** return a successful status code and the id
+ `POST` `/user`, **SHOULD** return a successful status code and an empty string

Regards,

**Comment by jaguililla** [comment]
Absolutely... Go ahead, I will review those routes to assure they behave as you expect,  thanks for inviting me to join your benchmark :)
